src/gemm_v2.cpp

```diff
@@ -324,41 +325,216 @@ inline void ProfilingRecordStop(const Handle& handle, HipEventPtr& start, HipEve
+    inputs.alpha = &alpha;
+    inputs.beta  = &beta;
+
+    // TODO - bharriso - Need to pass down workspace size & pointer.
+    hipblaslt_ext::GemmPreference pref;
+    pref.setMaxWorkspaceBytes(0);
+
+    hipblaslt_ext::GemmEpilogue epilogue;
+    hipblaslt_ext::GemmTuning tuning;
+
+    hipblasOperation_t opTypeA = gemm_desc.transA ? HIPBLAS_OP_T : HIPBLAS_OP_N;
+    hipblasOperation_t opTypeB = gemm_desc.transB ? HIPBLAS_OP_T : HIPBLAS_OP_N;
+
+    hipblaslt_ext::GemmProblemType problemType;
+    problemType.op_a         = opTypeA;
+    problemType.op_b         = opTypeB;
+    problemType.type_a       = hip_type_AB;
+    problemType.type_b       = hip_type_AB;
+    problemType.type_c       = hip_type_C;
+    problemType.type_d       = hip_type_C;
+    problemType.type_compute = HIPBLAS_COMPUTE_32F;
+
+    hipblaslt_ext::Gemm gemm(handle.HipblasLtHandle().get(),
+                             opTypeA,
+                             opTypeB,
+                             hip_type_AB,
+                             hip_type_AB,
+                             hip_type_C,
+                             hip_type_C,
+                             HIPBLAS_COMPUTE_32F);
+
+    check_hipblas_status(gemm.setProblem(gemm_desc.m,
+                                         gemm_desc.n,
+                                         gemm_desc.k,
+                                         gemm_desc.batch_count,
+                                         gemm_desc.lda,
+                                         gemm_desc.ldb,
+                                         gemm_desc.ldc,
+                                         gemm_desc.ldc,
+                                         gemm_desc.strideA,
+                                         gemm_desc.strideB,
+                                         gemm_desc.strideC,
+                                         gemm_desc.strideC,
+                                         epilogue,
+                                         inputs,
+                                         problemType));
+
+    std::vector<hipblasLtMatmulHeuristicResult_t> heuristic;
+    check_hipblas_status(gemm.algoGetHeuristic(1, pref, heuristic));
+
+    check_hipblas_status(gemm.initialize(heuristic[0].algo, tuning, nullptr, handle.GetStream()));
+
+    {
+        HipEventProfiler profiler(handle);
+        check_hipblas_status(gemm.run(handle.GetStream()));
+    }
+}
+
+static void call_miopen_hipblasLt_gemm(const miopen::Handle& handle,
+                                       const miopen::GemmDescriptor& gemm_desc,
+                                       ConstData_t A,
+                                       std::size_t a_offset,
+                                       ConstData_t B,
+                                       std::size_t b_offset,
+                                       Data_t C,
+                                       std::size_t c_offset)
+{
+    MIOPEN_LOG_FUNCTION("hipBLASLt");
```

Comment:
MIOPEN_LOG_FUNCTION is intended to be used **_only_** in API layers. It outputs not only function name but also values of its parameters, and combines logging to console and code annotation for ROC-tracer. Example of use:
https://github.com/ROCm/MIOpen/blob/e5b984a5cc39e07aef1d75246d3272d8066e7a91/src/rnn_api.cpp#L388-L398

MIOPEN_LOG_FUNCTION should not be used in this file.

If you need some indicator that a function is called, please use Info2 logging:
```C++
MIOPEN_LOGI2("");
```
Info2 logging is enabled by `export MIOPEN_LOG_LEVEL=6` or higher.

Reply:
Should I also remove using this from the equivalent rocBLAS calls?
Or leave it for now?

Reply:
@BrianHarrisonAMD This is not necessary, but I would appreciate that. Thanks.